packages/chain-mon/src/balance-mon/service.ts

```diff
@@ -6,7 +6,8 @@ import {
   validators,
 } from '@eth-optimism/common-ts'
 import { Provider } from '@ethersproject/abstract-provider'
-import { ethers } from 'ethers'
+import { BigNumber, ethers } from 'ethers'
+import Safe from '@eth-optimism/contracts-bedrock/forge-artifacts/IGnosisSafe.sol/IGnosisSafe.0.8.19.json'
```

Comment:
I recommend we bundle the JSON ABI file directly into this repo because the ABI never changes and it reduces the dep on contracts-bedrock, ie we would need to make sure to publish an npm package that contains that artifact and we are not currently doing that